src/skydb.ts

```diff
@@ -283,6 +289,100 @@ export async function setDataLink(
   await this.registry.setEntry(privateKey, entry, setEntryOpts);
 }
 
+/**
+ * Gets the raw registry entry data at the given public key and data key.
+ *
+ * @param this - SkynetClient
+ * @param publicKey - The user public key.
+ * @param dataKey - The data key.
+ * @param [customOptions] - Additional settings that can optionally be set.
+ * @returns - The entry data.
+ */
+export async function getEntryData(
+  this: SkynetClient,
+  publicKey: string,
+  dataKey: string,
+  customOptions?: CustomGetEntryOptions
+): Promise<EntryData> {
+  validateHexString("publicKey", publicKey, "parameter");
+  validateString("dataKey", dataKey, "parameter");
+  validateOptionalObject("customOptions", customOptions, "parameter", DEFAULT_GET_ENTRY_OPTIONS);
+
+  const opts = {
+    ...DEFAULT_GET_ENTRY_OPTIONS,
+    ...this.customOptions,
+    ...customOptions,
+  };
+
+  const { entry } = await this.registry.getEntry(publicKey, dataKey, opts);
+  if (!entry) {
+    return { data: null };
+  }
+  return { data: entry.data };
+}
+
+/**
+ * Sets the raw entry data at the given private key and data key.
+ *
+ * @param this - SkynetClient
+ * @param privateKey - The user private key.
+ * @param dataKey - The data key.
+ * @param data - The raw entry data to set.
+ * @param [customOptions] - Additional settings that can optionally be set.
+ * @returns - The entry data.
+ * @throws - Will throw if the length of the data is > 70 bytes.
+ */
+export async function setEntryData(
+  this: SkynetClient,
+  privateKey: string,
+  dataKey: string,
+  data: Uint8Array,
+  customOptions?: CustomSetJSONOptions
```

Comment:
`CustomSetJSONOptions` is correct right? or is that a copy paste error

Reply:
Yes, but now that I think of it, it may be better to introduce `CustomSetEntryDataOptions` instead.